src/main/resources/assets/voidscape/lang/uk_ua.json

```diff
@@ -56,6 +56,7 @@
   "entity.voidscape.null_servant.ichor": "Іхоровий слуга",
   "entity.voidscape.null_servant.astral": "Астарльний слуга",
   "entity.voidscape.null_servant_augment_block": "Блок",
+  "entity.voidscape.strange_pearl": "Дивна перлина",
```

Comment:
Are these verified from someone who actually knows the language? I don't want google translate or DeepL in these.

Reply:
Not verified. Charred Bone and "Strange Pearl" are both translated already as items, and the uk_ua translation of "brick" comes from Mojang.

Reply:
I wouldn't trust that logic when it comes to language.

Reply:
Strange Pearl is unchanged, it's the same exact text as the item form with the initial lang entry marked differently.

Charred Bone Brick i made sure to check in minecraft how they combined the first half (Charred Bone) and the second half (Brick). Works the way I put down from what I understand. If you want, I can push this again but without that entry for now.

Reply:
I'd much prefer that any language change comes from real translators.